.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Is required to move this file?

Reply:
No, I moved it because I'm putting all the files in this folder, but I can put it back, do you prefer it where it was before?

Reply:
Which is the usual place?

Reply:
You can put in the root folder or in a folder called .github. I was using both, so I decided to move everything to the .github folder